portal/component-tests/utilisation-report-service/confirmation.component-test.js

```diff
@@ -6,18 +6,21 @@ const render = pageRenderer(page);
 describe(page, () => {
   let wrapper;
   const reportPeriod = 'June 2023';
-  const paymentOfficerEmail = 'tradefinance@barclays.com';
+  const paymentOfficerEmails = ['tradefinance1@barclays.com', 'tradefinance2@barclays.com'];
 
   beforeEach(() => {
-    wrapper = render({ reportPeriod, paymentOfficerEmail });
+    wrapper = render({ reportPeriod, paymentOfficerEmails });
   });
 
   it('should render page heading', () => {
     wrapper.expectText('[data-cy="confirmation-main-heading"]').toRead(`${reportPeriod} GEF report sent to UKEF`);
   });
 
   it('should render paragraph', () => {
-    wrapper.expectText('[data-cy="paragraph"]').toRead(`A confirmation email has been sent to ${paymentOfficerEmail}.`);
+    wrapper.expectText('[data-cy="paragraph"]').toRead('A confirmation email has been sent to:');
+    for (const paymentOfficerEmail of paymentOfficerEmails) {
+      wrapper.expectElement(`ul.govuk-list > li:contains("${paymentOfficerEmail}")`).toExist();
+  }
```

Comment:
nit: the `}` should be indented one level more

Reply:
ty